ban/core/versioning.py

```diff
@@ -272,6 +272,13 @@ def serialize(self, *args):
 
 class Redirect(db.Model):
 
+    __openapi__ = """
+        properties:
+            identifier_value:
```

Comment:
When you make a redirect, you have to give :
   . identifier name
   . ':'
   . value

So, what comes to me to define it was 'identifier_value'.
Another idea ?